code/game/machinery/suit_storage_unit.dm

```diff
@@ -60,6 +60,10 @@
 	storage_type = /obj/item/weapon/watertank/atmos
 
 /obj/machinery/suit_storage_unit/mining
+	suit_type = /obj/item/clothing/suit/hooded/explorer
+	mask_type = /obj/item/clothing/mask/gas
+
+/obj/machinery/suit_storage_unit/mining/eva

```

Comment:
You can add a single LAVALAND_ENABLED define and check for it. It would be really good if all it had to disable lavaland was commenting out a single define.